k8s/scripts/sysbox-deploy-k8s.sh

```diff
@@ -474,9 +475,12 @@ function install_sysbox_deps() {
 		elif semver_ge $kversion 5.17 && semver_lt $kversion 5.18; then
 			echo "Kernel version $kversion is 5.17"
 			cp -r "/opt/shiftfs-k5.17" "$host_run/shiftfs-dkms"
-		else
-			echo "Kernel version $kversion is >= 5.18"
+		elif semver_ge $kversion 5.18 && semver_lt $kversion 5.19; then
```

Comment:
Please update these if-statements per the latest version of [this](https://github.com/toby63/shiftfs-dkms#overview-of-branchesversions) table. We need something like this:

```
               elif semver_ge $kversion 5.18 && semver_lt $kversion 6.1; then
       			echo "Kernel version $kversion is >= 5.18 and < 6.17"
			cp -r "/opt/shiftfs-k5.18" "$host_run/shiftfs-dkms"

               elif semver_ge $kversion 6.1 && semver_lt $kversion 6.3; then
       			echo "Kernel version $kversion is >= 6.1 and < 6.3"
			cp -r "/opt/shiftfs-k5.18" "$host_run/shiftfs-dkms"

		else
			echo "Kernel version $kversion is >= 6.3"
			# dont copy shiftfs, but still proceed to install other deps
```

Reply:
I did see that upstream has support for 6.1/6.2, but the branch was not present on the [nestybox/shiftfs-dkms](https://github.com/nestybox/shiftfs-dkms) fork that is pulled in for the daemonset, so I omitted it here. Would we want to use upstream's branch for now until your fork pulls it in?

Reply:
You're right, the fork didn't have this latest branch. I just went ahead and pulled it. To avoid any inconsistency with our current installation method, please keep on using our fork.